src/blaze/system.clj

```diff
@@ -253,6 +261,13 @@
   (d/connect uri))
 
 
+(defmethod ig/init-key :authentication
+  [_ {:openid-provider/keys [url]}]
+  (let [jwks-json  (when (not-empty url) (authentication/jwks-json url))
```

Comment:
I would write:

```clojure
(if (str/blank? url)
    identity
    (do (log/info "Enabled authentication using OpenID provider:" url)
        (-> url
            authentication/jwks-json
            authentication/public-key
            authentication/wrap-authentication)))
```

Returning `identity` in case we have no url is the no-op middleware.